display.go

```diff
@@ -0,0 +1,116 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"image"
+	r "reflect"
+	"strings"
+
+	"github.com/cosmos72/gomacro/imports"
+)
+
+// Support an interface similar - but not identical - to the IPython (canonical Jupyter kernel).
+// See http://ipython.readthedocs.io/en/stable/api/generated/IPython.display.html#IPython.display.display
+// for a good overview of the support types. Note: This is missing _repr_markdown_ and _repr_javascript_.
+
+var globalReceipt *msgReceipt // ugly global variable. any alternative?
```

Comment:
I think restricting this to just rendering (returning a mime bundle) and pulling the actual display portion will clean this up a lot.